### PR TITLE
Add Specific Instructions for Svelte Kit

### PR DESCRIPTION
**Pull request by alteredorange**
Get windi working with the latest svelte kit.

### CODE REVIEW

File: integrations/svelte.md [hunk 1]
```diff
@@ -29,6 +29,28 @@ Add `"svelte.plugin.css.diagnostics.enable": false` to your VS Code configuratio
 
 Add <kbd>[svelte-windicss-preprocess]</kbd> to your bundler configuration.
 
+### Svelte Kit (As of 1.0.0-next.100)
```
Comment (alexanderniebuhr) [inline integrations/svelte.md:32]: I would prefer if the instruction part is actually moved down under the vanilla Svelte section please

File: integrations/svelte.md [hunk 2]
```diff
@@ -29,6 +29,28 @@ Add `"svelte.plugin.css.diagnostics.enable": false` to your VS Code configuratio
...
+You should only need to install the vite plugin, not the svelte-preprocess plugin - (`npm i -D vite-plugin-windicss`). Then add the plugin to your svelte config. Don't forget to add `import "virtual:windi.css"` to the top of your $layout.svelte!
```
Comment (alexanderniebuhr) [inline integrations/svelte.md:33]: Change to something like, you also should include an very basic example of  `$layout.svelte` file

Svelte Kit uses vite as a bundler, therefore we suggest to use our [vite plugin](https://windicss.org/integrations/vite.html)
Add the plugin to your svelte config. 
After that add `import "virtual:windi.css"` to the top of your $layout.svelte file.

File: integrations/svelte.md [hunk 3]
```diff
@@ -29,6 +29,28 @@ Add `"svelte.plugin.css.diagnostics.enable": false` to your VS Code configuratio
...
+import preprocess from 'svelte-preprocess'
+
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	preprocess: preprocess(),
+	kit: {
+		target: '#svelte',
+		vite: () => ({
+				plugins: [
+				WindiCSS.default()
```
Comment (alexanderniebuhr) [inline integrations/svelte.md:46]: the import of WindiCSS npm package seems to be missing

**Review by alexanderniebuhr — CHANGES_REQUESTED** [review]
thanks for you PR. I have reviewed it and marked things to change for you

**Review by alteredorange — COMMENTED** [review]
Sounds good, I'll add those changes!

**Review by alexanderniebuhr — APPROVED** [review]
/lgtm @hannoeru

**Pull request merged**